.github/workflows/update-pgxn-version.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Version on PGXN Config Files
+
+env:
+  PROJECT_VERSION: ${{ github.event.inputs.project_version }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      project_version:
+        description: "Version number to be released e.g 10.0.3"
+
+jobs:
+  update_pgxn:
+    name: Update pgxn configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch issue_154 https://github.com/citusdata/tools.git
```

Comment:
Needs to be updated after https://github.com/citusdata/tools/pull/155 gets merged.

Reply:
Fixed